components/BrSetupReviewSection.vue

```diff
@@ -30,5 +32,27 @@ export default {
   }
 };
 </script>
-<style>
+<style lang="scss">
+$breakpoint-sm: 767px;
+$breakpoint-xs: 320px;
+
+@mixin mobile {
+  @media (min-width: #{$breakpoint-xs}) and (max-width: #{$breakpoint-sm}) {
```

Comment:
This custom CSS seems like it might be brittle -- is there a better way to do it via quasar CSS classes or with whatever quasar suggests? If not, we can roll with it for now, but we should expect it to break and need additional maintenance.

Reply:
Looks like we are going to have to roll with this for now. I can't do what I need via Quasar CSS classes.